### PR TITLE
[improve][doc] add a new metric (pulsar_topic_load_times)

### PR DESCRIPTION
**Pull request by Anonymitaet**
This PR adds docs for https://github.com/apache/pulsar/pull/20472

- [x] `doc`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
Preview looks good:
image

File: docs/reference-metrics.md [hunk 1]
```diff
@@ -140,7 +140,8 @@ All the broker metrics are labeled with the following labels:
 | pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena. |
 | pulsar_ml_cache_used_size | Gauge  | The size used to store the payloads of entries (in bytes). |
 | pulsar_ml_count | Gauge  | The number of currently opened managed ledgers. |
-| topic_load_times | Summary | The topic load latency calculated in milliseconds. |
+| ~~topic_load_times (deprecated)~~ | ~~Summary~~ | ~~The topic load latency calculated in milliseconds.~~ |
```
Comment (Anonymitaet) [inline docs/reference-metrics.md:143]: I add strikethroughs on purpose (to make it more prominent and allow users to be aware of it's deprecated).

**Comment by Anonymitaet** [comment]
@Technoboy- 
Could you please help review this PR? Thank you!

**Review by Technoboy- — APPROVED** [review]

**Comment by Technoboy-** [comment]
> @Technoboy- Could you please help review this PR? Thank you!

👍🏻

**Pull request merged**